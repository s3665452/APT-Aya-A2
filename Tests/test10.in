BBLRUBYRYLUYRYBRBULBRYUYYLRRUURULYYURBLLBRUBYYLURYYLYBBRUUBLYRYUYBLURLBYYURBLLBUYRBLBULRBLBURLRLURLU
Ain
Mich
turn 1 B 2
turn 3 Y 2
turn 4 R 4
turn 5 B 4
turn 2 R 4
turn 0 B 4
turn 0 R 4
turn 0 L 3
turn 0 U 3
turn 0 Y 5
turn 2 R 2
turn 5 L 2
turn 4 Y 5
turn 1 Y 3
turn 0 Y 5
turn 0 U 5
turn 0 R 4
turn 3 U 5
turn 0 L 1
turn 0 B 1
turn 0 R 4
turn 4 Y 3
turn 3 Y 4
turn 0 B 5
turn 2 Y 4
turn 1 B 5
turn 0 R 5
turn 0 U 2
turn 5 L 1
turn 0 B 5
turn 0 U 2
turn 0 L 6
turn 2 Y 3
turn 1 Y 3
turn 3 R 4
turn 0 U 4
turn 5 L 5
turn 0 U 4
turn 4 Y 3
turn 0 R 5
turn 0 L 5
turn 0 U 4
turn 0 B 2
turn 0 Y 1
turn 5 L 4
turn 3 L 5
turn 4 R 3
turn 0 U 4
turn 0 B 5
turn 2 U 4
turn 0 L 4
turn 0 R 3
turn 0 B 5
turn 1 R 3
turn 0 B 5
turn 0 L 2
turn 0 Y 6
turn 1 B 5
turn 3 B 3
turn 0 R 3
turn 4 U 5
turn 5 Y 2
turn 0 U 5
turn 0 R 1
turn 2 L 2
turn 0 Y 6
turn 0 R 1
turn 0 L 6
turn 3 B 4
turn 4 Y 4
turn 5 R 2
turn 2 U 1
turn 0 Y 5
turn 0 U 3
turn 1 Y 5
turn 0 U 6
turn 0 L 3
turn 2 R 2
turn 0 Y 4
turn 3 B 3
turn 1 R 5
turn 0 B 3
turn 5 R 5
turn 0 U 1
turn 4 Y 1
turn 0 L 5
turn 0 Y 6
turn 0 R 6
turn 2 L 4
turn 1 L 5
turn 4 L 4
turn 0 U 6
turn 5 B 1
turn 3 B 6
turn 0 R 6
turn 0 Y 6
turn 0 U 6




