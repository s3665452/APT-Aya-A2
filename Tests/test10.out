Factories:
0: F
1: U U U R
2: L L B B
3: B Y Y Y
4: Y U U U
5: R B B U

Score for Player Ain: 79
Mosaic for Ain:
1:         . || B Y R U L
2:       . . || L B Y R U
3:     . . . || U L B Y R
4:   . . . . || R U L B Y
5: . . . . . || Y R U L B
broken:

Score for Player Mich: 96
Mosaic for Mich:
1:         . || B Y R U L
2:       . . || L B Y R U
3:     . . . || U L B Y R
4:   . . . . || R U L B Y
5: . . . . . || Y R U L B
broken: